rinja_parser/src/node.rs

```diff
@@ -1268,25 +1270,22 @@ fn is_rust_keyword(ident: &str) -> bool {
     const KW7: &[[u8; MAX_KW_LEN]] = &[*b"unsized_", *b"virtual_"];
     const KW8: &[[u8; MAX_KW_LEN]] = &[*b"abstract", *b"continue", *b"override"];
 
-    const KWS: &[&[[u8; MAX_KW_LEN]]] = &[KW0, KW1, KW2, KW3, KW4, KW5, KW6, KW7, KW8];
-
-    // Ensure that all strings are ASCII, because we use `from_utf8_unchecked()` further down.
-    const _: () = {
-        let mut i = 0;
-        while i < KWS.len() {
-            let mut j = 0;
-            while KWS[i].len() < j {
-                let mut k = 0;
-                while KWS[i][j].len() < k {
-                    assert!(KWS[i][j][k].is_ascii());
-                    k += 1;
-                }
-                j += 1;
+    &[KW0, KW1, KW2, KW3, KW4, KW5, KW6, KW7, KW8]
+};
+
+/// Ensure that all strings are UTF-8, because we use `from_utf8_unchecked()` further down.
+#[test]
+fn ensure_utf8() {
```

Comment:
Is there a need to duplicate these consts between the two crates? Can't we make them public behind `#[doc(hidden)]` instead?

Reply:
> Is there a need to duplicate these consts between the two crates?

Sadly, I think yes. The list items are not the same with `loop self Self` missing from the generator list.

Reply:
Then why not add another list for these 3 keywords in `rinja_parser`? Having some much code duplicated is itching me strongly. ^^'

If you want I don't mind doing it. In this case just merge this PR and I'll take a look.

Reply:
> In this case just merge this PR and I'll take a look.

Thanks! Sure, go ahead. :)